contracts/near-campaign/src/refund_keys.rs

```diff
@@ -19,17 +20,14 @@ impl Campaign {
         "Cannot refund inactive or non-existing key"
       );
 
-      self.keys.insert(&key, &KeyStatus::Refunded);
-      self.keys_stats.active -= 1;
-      self.keys_stats.refunded += 1;
-
-      if self.keys_stats.active == 0 {
-        self.status = CampaignStatus::Completed;
-      };
-
-      Promise::new(env::current_account_id())
-        .delete_key(key)
-        .then(Promise::new(beneficiary_id.clone()).transfer(self.tokens_per_key));
+      Promise::new(beneficiary_id.clone())
+        .transfer(self.tokens_per_key)
+        .then(ext_self::on_refunded_key(
+          key,
+          env::current_account_id(),
+          0,
+          BASE_GAS.mul(2), // 50 TGas
```

Comment:
Can we reduce the gas amount? The idea is to refund as many keys as we can per one call. With 50 Tgas price, we can refund only 4 (5?) keys per call

Reply:
please look at this:
https://github.com/NEAR-labs/contracts.near-linkdrop/pull/13/commits/69a3fae0967aedab6762d1dde07d4662f5a69d06